cli/command/system/info.go

```diff
@@ -172,16 +172,20 @@ func prettyPrintInfo(dockerCli *command.DockerCli, info types.Info) error {
 			fmt.Fprintf(dockerCli.Out(), "\n")
 		}
 		if len(info.SecurityOptions) != 0 {
+			kvs, err := types.DecodeSecurityOptions(info.SecurityOptions)
+			if err != nil {
+				return err
+			}
```

Comment:
I'm pretty sure this would fail if you make a request to a Docker 1.12.x daemon with a newer client since the older server will not have encoded the security options in any special way. This could probably be fixed by adding a special check in the `DecodeSecurityOptions()` function.